packages/block-library/src/read-more/index.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Server-side rendering of the `core/read-more` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/read-more` block on the server.
+ *
+ * @param array    $attributes Block attributes.
+ * @param string   $content    Block default content.
+ * @param WP_Block $block      Block instance.
+ * @return string  Returns the post link.
+ */
+function render_block_core_read_more( $attributes, $content, $block ) {
+	if ( ! isset( $block->context['postId'] ) ) {
+		return '';
+	}
+
+	$post_ID            = $block->context['postId'];
+	$justify_class_name = empty( $attributes['justifyContent'] ) ? '' : "is-justified-{$attributes['justifyContent']}";
+	$wrapper_attributes = get_block_wrapper_attributes( array( 'class' => $justify_class_name ) );
+	$more_text          = ! empty( $attributes['content'] ) ? $attributes['content'] : __( 'Read more' );
+	return sprintf(
+		'<a %1s href="%2s" target="%3s">%4s</a>',
+		$wrapper_attributes,
+		get_the_permalink( $post_ID ),
+		esc_attr( $attributes['linkTarget'] ),
+		$more_text
```

Comment:
Can we "trust" this value not to need escaping @ntsekouras?

Reply:
I just spotted this too while doing a quick audit of dynamic blocks.

My understanding is that because this is a user supplied value and can contain html (formatting like bold/italic), it should be passed through `wp_kses`.

Here's a PR - https://github.com/WordPress/gutenberg/pull/38650

Reply:
Thanks for the PR @talldan!